config/samples/kustomization.yaml

```diff
@@ -1,3 +1,3 @@
 ## This file is auto-generated, do not modify ##
 resources:
-- koku-metrics-cfg_v1beta1_kokumetricsconfig.yaml
+- costmanagement-metrics-cfg_v1beta1_costmanagementmetricsconfig.yaml
```

Comment:
does this file exist?

Reply:
It does now - I added copying it to the correct name as part of the `make upstream` command